Runtime/Scripts/Qonversion.cs

```diff
@@ -382,6 +393,20 @@ private void OnEligibilities(string jsonString)
             EligibilitiesCallback = null;
         }
 
+        // Called from the native SDK - Called when deferred or pending purchase occured
+        private void OnUpdatedPurchases(string jsonString)
+        {
+            OnUpdatedPurchasesReceived purchasesListener = UpdatedPurchasesOccurred;
+            if (purchasesListener == null)
+            {
+                return;
+            }
+
+            Debug.Log("OnUpdatedPurchases " + jsonString);
+            Dictionary<string, Permission> permissions = Mapper.PermissionsFromJson(jsonString);
+            purchasesListener(permissions);
```

Comment:
Can we use this syntax? I'd taken it from [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/event)

```suggestion
            Debug.Log("OnUpdatedPurchases " + jsonString);
            Dictionary<string, Permission> permissions = Mapper.PermissionsFromJson(jsonString);
            UpdatedPurchasesOccurred?.invoke(permissions);
```

Reply:
The only thing I hesitate is that `.?` was added since C# 8.0. The clients with lower C# version may have problems during compilation. Also, I do not want to print log without listener.
https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references 
By the way, my implementation is from the following example:
https://docs.microsoft.com/en-us/dotnet/standard/events/how-to-raise-and-consume-events#example-3
What do you think?

Reply:
Clear. Let's leave it as is then)